addons/im_livechat/static/src/legacy/models/public_livechat_message.js

```diff
@@ -260,7 +261,7 @@ const PublicLivechatMessage = Class.extend({
      * @return {boolean}
      */
     _isMyselfAuthor() {
-        return this.hasAuthor() && (this.getAuthorID() === session.partner_id);
+        return this.hasAuthor() && (this.getAuthorID() === this.currentPartnerId);
```

Comment:
Just compare with `this.messaging.publicLivechatGlobal.options.current_partner_id` (don't define `this.currentPartnerId`)

Reply:
Done